features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -348,6 +348,46 @@ Feature: Gift Exchange Challenge
     Then I should see "myname1"
       And I should see "Fulfilled Story"
 
+  Scenario: A mod can default all incomplete assignments
+    Given everyone has their assignments for "Awesome Gift Exchange"
+      And I am logged in as "myname1"
+      And I fulfill my assignment
+   When I am logged in as "mod1"
+     And I go to the "Awesome Gift Exchange" assignments page
+     And I follow "Default All Incomplete"
+   Then I should see "All unfulfilled assignments marked as defaulting."
+     And I should see "Undefault myname2"
+     And I should see "Undefault myname3"
+     And I should see "Undefault myname4"
+
+  Scenario: User can default and undefault on their assignment 
+   Given everyone has their assignments for "Awesome Gift Exchange"
+   When I am logged in as "myname1"
+     And I go to my user page
+     And I follow "Assignments"
+     And I follow "Default"
+   Then I should see "We have notified the collection maintainers that you had to default on your assignment."
+   When I am logged in as "mod1"
+     And I go to the "Awesome Gift Exchange" assignments page
+     And I check the 1st checkbox with id matching ".*"
+     And I press "Submit"
+   Then I should see "Assignment updates complete!"
+     And I should not see "Undefault"
+      When I am logged in as "myname1"
+     And I go to my user page
+     And I follow "Assignments"
+     And I follow "Default"
+   Then I should see "We have notified the collection maintainers that you had to default on your assignment."
+   When I am logged in as "mod1"
+     And I go to the "Awesome Gift Exchange" assignments page
+     And I fill in the 2nd field with id matching ".*" with "nonexistent"
```

Comment:
I gave it a try, and this should work just fine as `And I fill in "Pinch Hitter:" with "nonexistent"`. If it doesn't, though, it would help to add a comment above so we can understand what's being done here.